wallet/lib/pages/settings/screens/general_screen/screens/payment_screen/widgets/transaction_card.dart

```diff
@@ -123,13 +123,13 @@ class LeadingBuilder extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     switch (transactionType) {
-      case TransactionType.SEND:
+      case WalletHistoryTransactionType.SEND:
         return onSendReceive(context);
```

Comment:
Code reusability would be better as there are multiple copy pastes are here.

Reply:
Yeah, You are right. I have combined the similar switch cases now.